js/index.js

```diff
@@ -1,10 +1,25 @@
 $(document).ready(function() {
 
-  $('a').click(function(){
+  $('.scroll-down').click(function(){
       $('html, body').animate({
           scrollTop: $( $.attr(this, 'href') ).offset().top
       }, 1200);
```

Comment:
this jitters pretty bad for me -- two things you can do to help it out: 
1. use velocity 
2. that main cover image you're using is a HUGE file. try both shrinking it down just a little, and using a jpg instead of a png -- a high quality jpg will look just as good, and jpg is a much better file format for storing photographic images (as opposed to things like logos)